tests/device_tests/test_session_id_and_passphrase.py

```diff
@@ -107,8 +107,8 @@ def test_clear_session(client):
     assert session_id == new_session_id
 
     # by clearing the session, the id is lost
-    response = client.call(messages.ClearSession())
-    assert isinstance(response, messages.Success)
+    response = client.call(messages.Initialize())
+    assert isinstance(response, messages.Features)
```

Comment:
again please drop the `clear_session` testcases completely, ISTM that all the cases are already covered in other tests